### PR TITLE
Upgrade RuboCop

### PR DESCRIPTION
**Pull request by bquorning**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -75,17 +84,14 @@ Style/FormatString:
 Style/GuardClause:
   Enabled: false
 
-Style/HashSyntax:
-  EnforcedStyle: ruby19
```
Comment (grosser) [inline .rubocop.yml]: wut ?

File: .rubocop.yml [hunk 2]
```diff
@@ -75,17 +84,14 @@ Style/FormatString:
...
-
 Style/IfUnlessModifier:
   Enabled: false
 
 Style/IndentHash:
   EnforcedStyle: consistent
 
-Style/IndentationWidth:
-  Width: 2
```
Comment (grosser) [inline .rubocop.yml]: keep this ?

Comment (bquorning) [inline .rubocop.yml]: No need to duplicate the default configuration: https://github.com/bbatsov/rubocop/blob/v0.48.1/config/default.yml#L651-L653

File: .rubocop.yml (hunk 1 above)
Comment (bquorning) [inline .rubocop.yml]: No need to duplicate the default configuration: https://github.com/bbatsov/rubocop/blob/v0.48.1/config/default.yml#L620-L621

**Comment by grosser** [comment]
nice!

On Thu, Apr 6, 2017 at 10:46 AM, Benjamin Quorning <notifications@github.com

**Pull request merged**